README.md

```diff
@@ -27,13 +27,19 @@ We provide examples in Swift and Objective-C. Run `carthage update --platform io
 
 #### [CocoaPods](https://cocoapods.org/)
 
-You'll need to install two pods, `MapboxNavigation.swift` and `MapboxDirections.swift`
-
 Add the following lines to your Podfile:
 
 ```ruby
-pod 'MapboxDirections.swift', :git => 'https://github.com/mapbox/MapboxDirections.swift.git', :commit => 'ceaf58b780fc17ea44a9150041b602d017c1e567'
-pod 'MapboxNavigation.swift', :git => 'https://github.com/mapbox/MapboxNavigation.swift.git', :tag => 'v0.0.4'
+# Required
+pod 'MapboxDirections.swift', :git => 'https://github.com/mapbox/MapboxDirections.swift.git', :commit => '955970285f7dee31b2c130023d7b383be51ab190'
+pod 'MapboxNavigation.swift', :git => 'https://github.com/mapbox/MapboxNavigation.swift.git', :commit => 'c5a637a201a3043c24e52f3e2c50a1657a8ffced'
+pod 'MapboxNavigationUI.swift', :git => 'https://github.com/mapbox/MapboxNavigation.swift.git', :commit => 'c5a637a201a3043c24e52f3e2c50a1657a8ffced'
+pod 'OSRMTextInstructions', :git => 'git@github.com:Project-OSRM/osrm-text-instructions.swift.git', :commit => 'bdd962bf5593f951bbbbd26435f732a829dacda7'
+pod 'AWSPolly', '~> 2.5'
+pod 'SDWebImage', '~> 4.0'
```

Comment:
AWSPolly and SDWebImage are already listed as dependencies in https://github.com/mapbox/MapboxNavigation.swift/blob/c5a637a201a3043c24e52f3e2c50a1657a8ffced/MapboxNavigationUI.swift.podspec#L51. That’s the correct way to specify a dependency. If this library requires a certain version of AWSPolly, say, that line should read `s.dependency "AWSPolly", "~> 2.5"`.

The only reasons we’d be listing a dependency _here_ are if:

1. We haven’t gotten around to pushing a podspec to CocoaPods.org (as in the case of MapboxNavigation.swift, MapboxNavigationUI.swift, and OSRMTextInstructions).
1. We want to pin to a specific commit (which is a bug – we should always pin to a tagged release).

Listing a dependency redundantly here is a hack that relies on undocumented CocoaPods behavior.